crypto/dilithium/p_dilithium3.c

```diff
@@ -0,0 +1,109 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <openssl/evp.h>
+#include <openssl/err.h>
+#include <openssl/mem.h>
+
+#include "../fipsmodule/evp/internal.h"
+#include "../evp_extra/internal.h"
+#include "sig_dilithium.h"
+
+static int pkey_dilithium3_keygen(EVP_PKEY_CTX *ctx, EVP_PKEY *pkey) {
```

Comment:
@[andrewhop](https://github.com/andrewhop) andrewhop [on Oct 4](https://github.com/awslabs/aws-lc/pull/637/files#r987414686)
NP: missing return check, looking at sign.c this function can't fail, but we should be consistent.

Reply:
fixed in [46bdba8](https://github.com/awslabs/aws-lc/pull/674/commits/46bdba84a5986bb71374e0f3e82805b0ece09124)